### ISSUE TITLE
Release new version ?

### ISSUE THREAD

**Issue by touilleMan** [hit9/img2txt#14]
Hi, thanks for this really cool tool !

I've noticed the version available on Pypi doesn't contains the range/xrange fix (and so is not compatible with Python 3). Could you release a new version to fix this ?

I have to admit I fix this by myself when trying the tool, then when I was about to send a PR about this, I realised it was already fixed in the repo ;-)

**Comment by dat-boris** [comment]
A workaround to install directly from the repo until the new release.

```
pip install git+https://github.com/hit9/img2txt.git --upgrade
```

**Comment by hit9** [comment]
Sorry, but I just released a new version (v2.4) to pypi.

**Comment by touilleMan** [comment]
Thx ;-)
